app/mainAppWindow/index.js

```diff
@@ -209,9 +208,9 @@ function createNewBrowserWindow(windowState) {
 		height: windowState.height,
 		backgroundColor: isDarkMode ? '#302a75' : '#fff',
 
-		show: false,
+		show: true,
```

Comment:
not sure if we want to revert this as it will show the screen loading. But don't really fully remember why we did add it TBH